package.json

```diff
@@ -44,7 +45,7 @@
     "eslint": "^8.2.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "husky": "^7.0.4",
-    "jsdoc": "^3.6.7",
+    "jsdoc": "^4.0.0",
```

Comment:
I'm a bit scared this might break things. V4 is new and they don't have documentation for it. Is there a problem between jsdoc 3 and Node 18? It seemed to be working fine for me 🤔

Reply:
It's working fine. just gave me a vulnerability warning. But I think we can wait until we have documentation for it and see if it has breaking changes. gonna roll it back to "^3.6.7" since it's safer
<img width="941" alt="Screen Shot 2023-02-02 at 6 31 20 PM" src="https://user-images.githubusercontent.com/58053149/216384146-7e1351a2-38d8-4ef1-b9f3-35de1e934411.png">